app/uk/gov/hmrc/exports/services/reversemapping/DeclarationId.scala

```diff
@@ -14,6 +14,6 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.exports.services.reversemapping.declaration
+package uk.gov.hmrc.exports.services.reversemapping
 
-case class XmlParsingException(message: String) extends RuntimeException(message)
+case class DeclarationId(eori: String, mrn: String)
```

Comment:
Think this name is misleading. MRN is a declarationId, eori is a userId. Should be called 'context' or something like that imo.